runtime/src/lib.rs

```diff
@@ -595,6 +596,13 @@ impl_runtime_apis! {
 		}
 	}
 
+
+	impl pallet_encointer_ceremonies_rpc_runtime_api::CeremoniesApi<Block, AccountId> for Runtime {
+		fn get_reputations() -> Vec<(CommunityCeremony, AccountId, Reputation)> {
+			EncointerCeremonies::get_reputations()
```

Comment:
Why plural? Does this return reputation for all accounts? Why?

Reply:
after discussion with @clangenb I believe we should merge this as is and accept bad performance for now. The solution will be slightly more involved than I thought:
https://github.com/encointer/pallets/issues/118